### DISCUSSION TITLE
nvim-treesite does not load your commands

### DISCUSSION THREAD

**Discussion by astahjmo** [nvim-tree/nvim-tree.lua#1423] [Q&A] [answered]
I am using lua to do my nvim configuration, using packer I am trying to configure nvim-tree like this

```lua
use {
  'kyazdani42/nvim-tree.lua',
  requires = {
    'kyazdani42/nvim-web-devicons', -- optional, for file icons
  },
  config = function() require("nvim-tree").setup()
}
  ```
  and it doesn't work, but if I call the function "setup" from anywhere in the code (example: if I call it from init.lua ) it works. how can I make this works calling the setup from packer ?

**Comment by kyazdani42** [comment, marked as answer]
@astahjmo i answered you on matrix, but you forgot to close the function with an `end` keyword
